src/main/scala/com/holdenkarau/spark/validator/HistoricData.scala

```diff
@@ -1,3 +1,87 @@
 package com.holdenkarau.spark.validator
 
-case class HistoricData(jobid: Long, counters: scala.collection.Map[String, Long])
+import java.sql.Timestamp
+
+import org.apache.spark.sql.types.{LongType, StringType, StructField, StructType}
+import org.apache.spark.sql.{DataFrame, Row, SQLContext}
+
+case class HistoricData(counters: scala.collection.Map[String, Long]) {
+  /**
+   * Saves historic data to the given path.
```

Comment:
maybe (for our own reference) add a note that the path should already have the date in it.